freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -197,10 +204,10 @@ def handle(self, event):
             # Fetch buildinfo
             buildinfo = koji_api.get_build(bundle_nvr)
             related_images = (
-                buildinfo.get("extra", {})
-                .get("image", {})
-                .get("operator_manifests", {})
-                .get("related_images", {})
+                buildinfo.get('extra', {})
```

Comment:
Formatting fixes should be in separate commits, but I'll let this one slide :smile:

Reply:
Oh ok, sure, next time I will do it separately.